### ISSUE TITLE
Image distort on zoom visualizer crop Vertical and Horizontal but preview work fine

### ISSUE THREAD

**Issue by blenderdeluxe** [dai-siki/vue-image-crop-upload#205]
Hi!, 

I installed and all its great but the visualizer image on zoom slider distort the image only on visualizer not on preview I show you a example how can fix it? I used Chrome

![Screenshot_4](https://user-images.githubusercontent.com/7284321/74551039-3afef280-4f31-11ea-8799-700fb3b189ea.png)

**Comment by georgejoseph1994** [comment]
I faced this issue and noticed there was a conflict in style. Images were having a style max-width:100% from tailwind css.
I fixed it by adding max-width:unset; to the image.

**Comment by Eboubaker** [comment]
Yeah it worked. In my css file i added an exception for the class `.vicp-img` (which is used by this package for the edit zone image)
```css
.vicp-img{
    max-width:unset;
}
```
